### DISCUSSION TITLE
I can't anymore clone reposity on terminal of Mac

### DISCUSSION THREAD

**Discussion by Fabietto1988** [community/community#102042] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hi, 
I can't anymore clone my reposity on terminal of Mac, always ask for user name and then password and doesn't work, I tried to make a Token but it doesn't even work somebody could help me please?
I study for be developers but today was difficult go forward

**Comment by ananuness** [comment]
Hii, Fabietto! Could you share more info about the error you're getting? After you inform your password, what happens?

**Comment by Fabietto1988** [comment]
Hi Anna thanks for answer soon, the is the message of error 

(base) fabioanfuso@MacBook-Pro-di-fabio documents % git clone https://github.com/Fabietto1988/K-Means.git
Cloning into 'K-Means'...
Username for 'https://github.com': Fabietto1988
Password for 'https://Fabietto1988@github.com': 
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/Fabietto1988/K-Means.git/'

I'am really confusion about that happening never I had this error always worked well my terminal with git, I tried to creare a token but dosen't work same!

**Reply by ananuness** [reply]
Oh!! It's because now it's recommended and as I see it's mandatory to have a personal access token now! You'll have to create one to use https URLs. [Here](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-fine-grained-personal-access-token) you can see the step by step to create one! Did you already follow these steps?

**Comment by Fabietto1988** [comment]
Yes, i already tried but doesn't work same...

**Reply by ananuness** [reply]
Could you describe what changes for you? And one more thing, have you thought about using SSH instead? It can be more practical!

**Reply by Fabietto1988** [reply]
before when I create the reposity the in the terminal clone never ask me password now yes. I'll try to using SSH but yesterday was reale frustration all this :)
Thanks a lot for your help

**Reply by ananuness** [reply]
Oh, yes, the password now is your access token! Sadly now it's the new way. I think you will like more to use SSH! As you can make it without a password! Give it a try! If you need help setting up an SSH key, I'm here.

**Reply by Fabietto1988** [reply]
Thanks Ana, could share me the link or step for make SSH? It's Easy or Hard do it? Im Junior developer I love this world but i'am humble I prefer ask :)

**Reply by ananuness** [reply]
Oh, it's easy, but there are some steps to do it, so I searched for a friendly tutorial for you! It starts at [How to generate SSH keys locally](https://kinsta.com/blog/generate-ssh-key/#how-to-generate-ssh-keys-locally) and goes to the end of the article! 

There are some explanations to it (that's why it looks long), but nothing to worry about! If you have any additional questions don't hesitate to ask 🙆‍♀️ and I'm a junior too hehe

**Reply by Fabietto1988** [reply]
Thank a lot, I'll let you know soon, now i'am study for a examen :)
